src/charmed_kubeflow_chisme/testing/cos_integration.py

```diff
@@ -97,25 +104,89 @@ async def deploy_and_assert_grafana_agent(
 
     # Note(rgildein, dnplas): The grafana agent charm will go to BlockedStatus if it is not
     # related to any consumer (e.g. prometheus-k8s, grafana-k8s).
-    await model.wait_for_idle(apps=[GRAFANA_AGENT_APP], status="blocked", timeout=5 * 60)
+    # Note(rgildein): We use an idle_period of 60 so we can be sure that the targets have already
+    # been scraped.
```

Comment:
This makes sense, but what if the charm sets a scrape time higher than the default 60s? Maybe we should set this value in the charm test itself?

Reply:
Let's add it, I think it will be easier to have it available now.